### PR TITLE
*/*: misc fixes

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.200.2506-r1.ebuild [hunk 1]
```diff
@@ -18,7 +18,7 @@ KEYWORDS="~amd64 ~x86"
 IUSE=""
```
Comment (arthurzam) [inline media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.200.2506-r1.ebuild:18]: empty `IUSE`

File: media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.300.2604.ebuild [hunk 2]
```diff
@@ -18,7 +18,7 @@ KEYWORDS="~amd64 ~x86"
 IUSE=""
```
[the remark this answers is not in the document]
Comment (thesamesam) [reply media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.300.2604.ebuild:18]: Thanks, you’re of course right, but I’m avoiding style changes here as these aren’t my packages

**Comment by a17r** [comment]
This is needlessly touching a lot of old and obsolete ebuilds that should be dropped en masse instead.

I agree with getting rid of `eutils` wherever there is an ebuild already at `EAPI-7` but otherwise in many places it is just churn.

**Comment by thesamesam** [comment]
The work is already done and was automated for the most part.

**Review by Polynomial-C — APPROVED** [review]
Okay for base-system and my packages.

**Comment by thesamesam** [comment]
Thanks @Polynomial-C, I was unsure about bothering given it was introducing some (false positive) CI noise, but thanks for noticing - so I went ahead. Cheers!

**Pull request closed without merging**